pnpm-lock.yaml

```
@@ -0,0 +1,18 @@
+lockfileVersion: '6.0'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
+devDependencies:
+  prettier:
+    specifier: 3.0.0
+    version: 3.0.0
+
+packages:
+
+  /prettier@3.0.0:
+    resolution: {integrity: sha512-zBf5eHpwHOGPC47h0zrPyNn+eAEIdEzfywMoYn2XPi0P44Zp0tSq64rq0xAREh4auw2cJZHo9QUob+NqCQky4g==}
+    engines: {node: '>=14'}
```

```suggestion
    engines: {node: '>=18'}
```
Shouldn't it be greater than >=18 as now we have moved to node 18.

---

And less than 19.0.0

---

This is just showing the node compatibility of the package.

---

lol that is true 🤦‍♂️